modules/Date.js

```diff
@@ -0,0 +1,7 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+export default function showDate() {
+  const now = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
+  const div = document.querySelector('.date');
+  div.innerHTML = now;
+}
```

Comment:
- [ ] Please make sure to use arrow syntax for functions. 
```
export const showDate = () => {
  const now = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
  const div = document.querySelector('.date');
  div.innerHTML = now;
}
```
And make sure to import it as  `import { showDate } from './Date.js';` in List.js and index.js files 😉

Reply:
Date.js file has a single function which is required to be the default export. eslint flags an error when a single function is not the default export. I, therefore, exported it as 
```
 const showDate = () => {
  const now = DateTime.now().toLocaleString(DateTime.DATETIME_MED);
  const div = document.querySelector('.date');
  div.innerHTML = now;
} 

export default showDate;
```
and imported it as 
```
 import showDate from './Date.js';
```

Reply:
Thanks for taking time to explain that, It is clear to me now why you decided to use function declaration 🙌🏻